### DISCUSSION TITLE
Can I get github education pack even if am in high school?

### DISCUSSION THREAD

**Discussion by ouallanacim06** [community/community#81614] [GitHub Education]
### Hi everyone,

now am in high school and I study in a public school, and it doesn't have an email or website is it ok if I upload certificate that I study in that school, but there's a problem that it's in Arabic and French

**Comment by FakriSouyo** [comment]
When you apply for the GitHub Education Pack, feel free to upload certificates or documents in languages other than English. If your school isn't listed, you can usually manually enter its full name. For more details, check out the GitHub Education website or contact their support directly.

**Reply by ouallanacim06** [reply]
can I ask you for github support email

**Reply by FakriSouyo** [reply]
reach out through the following link for additional support, [education request](https://support.github.com/request/education) hope this will help.

**Comment by Shahnawazk7944** [comment]
Yes, high school students are eligible for the GitHub Education Pack. As long as you meet the eligibility requirements and can provide the necessary documentation like your original school id card photo, you can apply for the pack, even if it's in Arabic or French.
